finalitygadget/finalitygadget.go

```diff
@@ -279,6 +279,11 @@ func (fg *FinalityGadget) GetBlockByHash(hash string) (*types.Block, error) {
 }
 
 func (fg *FinalityGadget) QueryTransactionStatus(txHash string) (*types.TransactionInfo, error) {
+	// Validate the tx hash is a valid EVM tx hash
+	if len(txHash) != 66 || txHash[:2] != "0x" {
+		return nil, fmt.Errorf("invalid EVM transaction hash")
+	}
```

Comment:
- How about we wrap this into a function?
- Is there any existing util function in Op / Eth dependencies?

Reply:
> Is there any existing util function in Op / Eth dependencies?

asked cursor in both codebases and there isn't any.

i will create a util function myself